src/site/content/en/blog/hid/index.md

```diff
@@ -354,6 +354,20 @@ where you can see all HID and USB device related events in one single place.
   <figcaption>Internal page in Chrome to debug HID.</figcaption>
 </figure>
 
+On most Linux systems, HID devices are mapped with read-only permissions by
+default. To allow Chrome to open a HID device, you will need to add a new [udev
+rule]. Create a file at `/etc/udev/rules.d/50-yourdevicename.rules` with the
+following content:
+
+```vim
+KERNEL=="hidraw*", ATTRS{idVendor}=="[yourdevicevendor]", MODE="0664", GROUP="plugdev"
+```
+
+where `[yourdevicevendor]` is `057e` if your device is a Nintendo Switch
```

Comment:
```suggestion
In the line above, `[yourdevicevendor]` is `057e` if your device is a Nintendo Switch
```

(To make it a complete sentence.)

Reply:
This is following `Create a file at `/etc/udev/rules.d/50-yourdevicename.rules` with the
following content:`

Reply:
I know. Like that both are complete sentences. I prefer this still rather than broken up sentences. But your call.

Reply:
I've edited post then. Thanks!